doc/source/advanced.md

```diff
@@ -199,3 +199,50 @@ To explicitly specify this strategy, use the following in your `config.yaml`:
 singleuser:
     schedulerStrategy: pack
 ```
+
+## Pre-pulling Images for Faster Startup
+
+When user images are of non-trivial sizes (as most of them are), **pre-pulling**
+the images on all nodes significantly decreases the amount of time it takes to
+start a user's server. Without pre-pulling, sometimes it can take upto 5-10
+minutes for a user's server to launch. With pre-pulling, it is cut down to a few
+seconds in most cases.
+
+By default, the user container image is pulled on to all nodes whenever a `helm
```

Comment:
With **pre-pulling**, which is enabled by default, the user's container image is pulled on all nodes...